Source/RealmSection.swift

```diff
@@ -33,9 +33,17 @@ class Section<T: Object> : NSObject {
     //MARK: Actions
     
     func insertSorted(object: T) -> Int {
+        self.insert(object)
+        self.sort()
```

Comment:
No need but this is just my style.
I love always seeing where I'm calling "self".
But there's your project, so okay.